package.json

```diff
@@ -29,40 +29,65 @@
     },
     "private": true,
     "dependencies": {
+        "@swc/helpers": "~0.5.0",
+        "axios": "^1.0.0",
         "msw": "^1.2.1",
         "tslib": "^2.5.2"
     },
     "devDependencies": {
         "@changesets/changelog-github": "^0.4.8",
         "@changesets/cli": "^2.26.1",
+        "@commander-js/extra-typings": "^11.0.0",
         "@nrwl/js": "16.2.1",
-        "@nx-dotnet/core": "^1.22.0",
-        "@nx/devkit": "16.2.1",
-        "@nx/esbuild": "^16.2.2",
+        "@nx-dotnet/core": "1.22.0",
+        "@nx/devkit": "16.5.5",
```

Comment:
The nx versions are all mismatched. Need to upgrade NX first, then add all the extra plugins as they should all match. 

Did you run the NX migrations for the devkit and esbuild plugins?

Reply:
No I think this is where I've gone wrong here

Reply:
Should have used user secrets

Reply:
Fixed